File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BasePaymentCashListItemType.cs

using System.Collections.Generic;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BasePaymentCashListItemType : BaseCoreCashListItemType
    {
        public string StatusCode { get; set; }
        public string TypeCode { get; set; }
        public int UserId { get; set; }
        public int StatusKey { get; set; }
        public int TypeKey { get; set; }
        public string UserName { get; set; }
        public List<int> MarkedTransKeys { get; set; }
        public BaseBankPaymentType Bank { get; set; }
        public BaseCreditCardType CreditCard { get; set; }
        public System.Collections.Generic.List<BasePaymentCashListItemTypePolicies> Policies { get; set; }
    }
}

File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BasePaymentCashListItemType.vb
Public Class BasePaymentCashListItemType
    Inherits BaseCoreCashListItemType

    Public Property StatusCode As String
    Public Property TypeCode As String
    Public Property UserId As Integer
    Public Property StatusKey As Integer
    Public Property TypeKey As Integer
    Public Property UserName As String
    Public Property Policies As List(Of BasePaymentCashListItemTypePolicies)
End Class
